src/js/common/schemaform/helpers.js

```diff
@@ -204,6 +204,24 @@ export function filterInactivePages(pages, form) {
   }, form.data);
 }
 
+export function stringifyForm(key, value) {
```

Comment:
Yeah, I agree. I'm going to change it to `stringifyFormReplacer`, since the mdn docs call that parameter a "replacer" function.